lms/djangoapps/certificates/tests/test_models.py

```diff
@@ -204,3 +206,39 @@ def test_no_double_default(self):
             ValidationError,
             BadgeImageConfiguration(mode='test2', icon=self.get_image('good'), default=True).full_clean
         )
+
+
+@attr('shard_1')
+class CertificateTemplateAssetTest(TestCase):
+    """
+    Test Assets are uploading/saving successfully for CertificateTemplateAsset.
+    """
+    def get_image(self, name):
+        """
+        Get one of the test images from the test data directory.
+        """
+        return ImageFile(open(TEST_DATA_ROOT / 'static' / name + '.jpg'))
+
+    def test_asset_file_saving_with_actual_name(self):
+        """
+        Verify that asset file is saving with actual name, No hash tag should be appended with the asset filename.
+        """
+        test_asset_1 = self.get_image('picture1')
```

Comment:
It seems to me `get_image` is useless here since we can pass file name like this `picture1.jpg` to `SimpleUploadedFile `. It should not be an absolute path.